orderapi/src/main/java/fi/hel/verkkokauppa/order/model/subscription/SubscriptionQueryBuilderBuilder.java

```diff
@@ -31,6 +30,11 @@ public QueryBuilder toQueryBuilder(SubscriptionCriteria criteria) {
 			qb.must(QueryBuilders.termQuery("namespace", criteria.getNamespace()));
 		}
 
+		if (criteria.getEndDateBefore() != null) {
```

Comment:
Should !StringUtils.isEmpty  check to be added like in others?

Reply:
These are DateTimes